tests/cupyx_tests/scipy_tests/fftpack_tests/test_fftpack.py

```diff
@@ -578,3 +579,45 @@ def test_irfft_overwrite(self, xp, scp, dtype):
         x = testing.shaped_random(self.shape, xp, dtype)
         return scp.fftpack.irfft(x, n=self.n, axis=self.axis,
                                  overwrite_x=True)
+
+
+@testing.parameterize(
+    {'shape': (32, 16, 4), 'data_order': 'F'},
+    {'shape': (4, 32, 16), 'data_order': 'C'},
+)
+class TestFftnView(unittest.TestCase):
+
+    @testing.for_complex_dtypes()
+    def test_fortran_ordered_view(self, dtype):
```

Comment:
```suggestion
    def test_contiguous_view(self, dtype):
```
And mention that the F-contiguous case was the issue.